library/pod-security-policy/seccomp/src_test.rego

```diff
@@ -1,33 +1,62 @@
 package k8spspseccomp
 
 test_input_seccomp_allowed_empty {
-    input := { "review": input_review, "parameters": input_parameters_empty}
+    input := { "review": input_review_pod_single, "parameters": input_parameters_empty}
     results := violation with input as input
-    count(results) > 0
+    count(results) == 1
 }
 
 test_input_seccomp_allowed_all {
-    input := { "review": input_review, "parameters": input_parameters_wildcard}
+    input := { "review": input_review_pod_single, "parameters": input_parameters_wildcard}
     results := violation with input as input
     count(results) == 0
 }
 
 test_input_seccomp_allowed_in_list {
-    input := { "review": input_review, "parameters": input_parameters_in_list}
+    input := { "review": input_review_pod_single, "parameters": input_parameter_in_list}
     results := violation with input as input
     count(results) == 0
 }
 
 test_input_seccomp_not_allowed_not_in_list {
-    input := { "review": input_review, "parameters": input_parameters_not_in_list}
+    input := { "review": input_review_pod_single, "parameters": input_parameters_not_in_list}
     results := violation with input as input
-    count(results) > 0
+    count(results) == 1
+}
+test_input_seccomp_pod_multiple_allowed_empty {
```

Comment:
allowed or not allowed? It looks like we are expecting 2 violations per the last line of this test